File: sportsbook-app/betting-service/src/main/java/com/sportsbook/bettingservice/controller/SampleController.java
package com.sportsbook.bettingservice.controller;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class SampleController {

    @GetMapping("/api/bettingservice/ping")
    public String ping() {
        return "betting-service pong";
    }
}

File: sportsbook-app/settlement-service/src/main/kotlin/com/sportsbook/settlementservice/controller/SampleController.kt
package com.sportsbook.settlementservice.controller

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class SampleController {

    @GetMapping("/api/settlementservice/ping")
    fun ping(): String = "settlement-service pong"
}
